### DISCUSSION TITLE
Why restrict login by GitHub only?

### DISCUSSION THREAD

**Discussion by iaxat** [zed-industries/zed#18060] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

Why the sign in option only is for GitHub users?
Why not expand it to GitLab, Codeberg, Gitea and other platforms?

**Comment by SomeoneToIgnore** [comment]
Consider asking generic questions in the discussions next time.

Nothing is really __restricted__ it's just not implemented yet: https://github.com/zed-industries/zed/issues/13296

Current reason for not expanding it further is low demand (as you can see from the number of 👍 on that issue) and other, more prioritized tasks to work on first.
